platform/android/java/app/src/com/godot/game/GodotApp.java

```diff
@@ -56,8 +64,36 @@ public class GodotApp extends GodotActivity {
 		}
 	}
 
+	// Shouldn't conflict with BaseGodotEditor.RUN_GAME_INFO etc
+	private final static int WINDOW_ID = 668;
+	private final static String TAG = GodotApp.class.getSimpleName();
```

Comment:
Can you move this to the top of the class (line 55) and replace the use of `GODOT` in the two logging statements in the static block with `TAG`